using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +244,111 @@ hypertable with a continuous aggregate.
 
 ---
 
+### Advanced Topics [](advanced-usage)
+
+#### Continuous Aggregates using Integer-Based Time [](create-integer)
+
+Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
+also possible to create your own custom scheme for handling
+aggregation for tables that are using an integer time
+column. This can be useful if you have tables that use other measures
+of time that can be represented as integer values, such as nanosecond
+epochs, minutes since founding date, or whatever is suitable for your
+application.
+
+As an example, suppose that you have a table with CPU and disk usage
+for some devices where time is measured in
+[microfortnights][fff-system]. Since you are using an integer-valued
+column as time, you need to provide the chunk time interval when
+creating the hypertable. In this case, let each chunk consist of a
```

Comment:
I guess you meant 1000 millifortnights, otherwise the sentence is repeating the first sentence:

```suggestion
creating the hypertable. In this case, let each chunk consist of 1000
```